lib/rubocop/cop/lint/unreachable_loop.rb

```diff
@@ -79,7 +79,7 @@ module Lint
       #   # bad
       #   2.times { raise ArgumentError }
       #
-      # @example AllowedPatterns: [/(exactly|at_least|at_most)\(\d+\)\.times/] (default)
+      # @example AllowedPatterns: ['(exactly|at_least|at_most)\(\d+\)\.times'] (default)
```

Comment:
No wonder users see the name Allowed**Patterns** and specify `/.../` instead of `'...'` for cases like this.